client/package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.27.2",
```

Comment:
I think we're using fetch elsewhere? Do we need axios too?

Reply:
no we don't need it, I was doing another tutorial and needed it fo that but I don't think it's going to effect anything with it being there?